packages/slate-react/src/utils/set-text-from-dom-node.js

```diff
@@ -1,5 +1,19 @@
 import findPoint from './find-point'
```

Comment:
Can you rename this to `setDomText`?

Reply:
Going to leave it as per your other comment. Thanks.